internal/renderers/femtovg/fonts.rs

```diff
@@ -572,12 +572,18 @@ pub(crate) fn layout_text_lines(
             font_height
         } else {
             // Note: this is kind of doing twice the layout because text_size also does it
-            font.text_size(
-                PhysicalLength::new(paint.letter_spacing()),
-                string,
-                if wrap { Some(max_width) } else { None },
-            )
-            .height_length()
+            let text_height = font
+                .text_size(
+                    PhysicalLength::new(paint.letter_spacing()),
+                    string,
+                    if wrap { Some(max_width) } else { None },
+                )
+                .height_length();
+            if elide && text_height > max_height {
+                font_height * (max_height.get() / font_height.get()).floor()
```

Comment:
Makes sense- thanks for looking into this. Isn't this expression just max_height?

Reply:
Calculating the actual height helps to get the correct vertical alignment for elided text. The difference becomes more significant with a larger font.

`max_height`:

![Screenshot 2023-09-18 at 19 33 40](https://github.com/slint-ui/slint/assets/140617/d87f595e-dcfb-4ffe-be55-6588b1aa4604) 

vs.

`font_height * (max_height.get() / font_height.get()).floor()`:

![Screenshot 2023-09-18 at 19 34 53](https://github.com/slint-ui/slint/assets/140617/a3448e67-e90a-4c3e-ad5d-c33fb593ac5f)

Reply:
Yeah, definitely. I added a clarifying comment and tweaked the code to make it easier to understand the intention